tests/e2e/specs/UndoRedo.spec.js

```diff
@@ -105,6 +105,13 @@ describe('Undo/redo', () => {
     const startEventMoveToPosition = { x: 300, y: 300 };
 
     cy.get('[data-test=undo]')
+      .click()
+      .should('be.disabled');
+
+    waitToRenderAllShapes();
+
+    cy.get('[data-test=redo]')
+      .click()
```

Comment:
These changes are needed as the undo button is no longer initially disabled. This is because we've added a start event, which can be undone by clicking the undo button.